index.js

```diff
@@ -1,6 +1,6 @@
 
 const defaultAlertSettings = {
-  version: "v1",
+  specVersion: 1,
```

Comment:
I can't comment on the specific line number inside this UI, but `operations[n].when.version` should become `operations[n].when.specVersion` too.